ElementX/Sources/Services/Room/RoomProxy.swift

```diff
@@ -139,7 +139,8 @@ class RoomProxy: RoomProxyProtocol {
         
     func addTimelineListener(listener: TimelineListener) -> Result<[TimelineItem], RoomProxyError> {
         let settings = RoomSubscription(requiredState: [RequiredState(key: "m.room.topic", value: ""),
-                                                        RequiredState(key: "m.room.canonical_alias", value: "")],
+                                                        RequiredState(key: "m.room.canonical_alias", value: ""),
+                                                        RequiredState(key: "m.room.join_rules", value: "")],
```

Comment:
I'm surprised the trailing `,` didn't trigger a warning 🤔
```suggestion
                                                        RequiredState(key: "m.room.join_rules", value: "")]
```

Reply:
it's a function argument